etc/config.xml

```diff
@@ -1,25 +1,32 @@
 <?xml version="1.0" ?>
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
-	<default>
-		<payment>
-			<vindi>
-				<active>0</active>
-				<model>Vindi\Payment\Model\Payment\Vindi</model>
-				<allow_installments>0</allow_installments>
-				<order_status>pending</order_status>
-				<title>Vindi</title>
-				<allowspecific>0</allowspecific>
-				<payment_action>authorize_capture</payment_action>
-				<group>offline</group>
-				<allowspecific>0</allowspecific>
-			</vindi>
-		</payment>
-		<vindiconfiguration>
-			<general>
-				<mode>2</mode>
-				<test>values</test>
-			</general>
-		</vindiconfiguration>
-	</default>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <payment>
+            <vindi>
+                <active>1</active>
+                <model>Vindi\Payment\Model\Payment\Vindi</model>
+                <allow_installments>0</allow_installments>
+                <order_status>pending</order_status>
+                <title>Vindi</title>
+                <allowspecific>0</allowspecific>
+                <payment_action>authorize_capture</payment_action>
+                <group>offline</group>
+            </vindi>
+            <vindi_bankslip>
+                <active>1</active>
+                <model>Vindi\Payment\Model\Payment\BankSlip</model>
+                <title>Vindi - Boleto Bancário</title>
+                <order_status>pending</order_status>
+                <allowspecific>0</allowspecific>
+                <payment_action>authorize</payment_action>
```

Comment:
Seria interessante colocar o payment_action como `purchase`, @laerte-guimaraes?

Reply:
Acredito que não tem um equivalente no Magento para isso..
Acho que o fluxo authorize and capture são o padrão do Magento 2.

Temos alguma alternativa nesse caso @GabrielRanghetti ?

Reply:
Pelo que busquei, como padrão, o Magento tem o payment_action `sale` para isso. Serviria?

Reply:
no config.xml do módulo, esse parâmetro `payment_action` aceita apenas _authorize_, _authorize_capture_ ou _capture_, que é qual função ele executa ao realizar um pedido, nas classes Model/Payment/BankSlip.php ou Model/Payment/Vindi.php.

Reply:
@GabrielRanghetti então, seguindo esse fluxo, quando for realizada a alteração de status ele vai usar um capture?

Reply:
Nesse caso, quando for disparado o webhook BillPaid ele vai criar a fatura no magento e mudar o status.

Reply:
Criar uma fatura no pedido (já existente) correto ? :thinking:

Reply:
Exato.